src/ALE/PLM_functions.F90

```diff
@@ -5,21 +5,177 @@ module PLM_functions
 
 implicit none ; private
 
-public PLM_reconstruction, PLM_boundary_extrapolation
+public PLM_boundary_extrapolation
+public PLM_extrapolate_slope
+public PLM_monotonized_slope
+public PLM_reconstruction
+public PLM_slope_wa
+public PLM_slope_cw
 
 real, parameter :: hNeglect_dflt = 1.E-30 !< Default negligible cell thickness
 
 contains
 
+!> Returns a limited PLM slope following White and Adcroft, 2008. [units of u]
+!! Note that this is not the same as the Colella and Woodward method.
+real elemental pure function PLM_slope_wa(h_l, h_c, h_r, h_neglect, u_l, u_c, u_r)
+  real, intent(in) :: h_l !< Thickness of left cell [units of grid thickness]
+  real, intent(in) :: h_c !< Thickness of center cell [units of grid thickness]
+  real, intent(in) :: h_r !< Thickness of right cell [units of grid thickness]
+  real, intent(in) :: h_neglect !< A negligible thickness [units of grid thickness]
+  real, intent(in) :: u_l !< Value of left cell [units of u]
+  real, intent(in) :: u_c !< Value of center cell [units of u]
+  real, intent(in) :: u_r !< Value of right cell [units of u]
+  ! Local variables
+  real :: sigma_l, sigma_c, sigma_r, u_min, u_max
```

Comment:
We should get into the habit of documenting the units of every real variable, and eventually enforce this for all code commits if possible.  In this case it would be easy as all have [units of u].

Reply:
Addressed in 553166ae8e8a871a0d1e39a091cbe707ef8a3159